build_keyboard.mk

```diff
@@ -204,7 +204,9 @@ else
 endif
 
 # User space stuff
-USER_PATH := users/$(KEYMAP)
+ifeq ("$(USER_PATH)","")
```

Comment:
(@jackhumbert) is this needed?

Reply:
Ah, nice!

Though, if you're adding this, it should be added to the documentation too, so that others know they can do this.

Reply:
Done, and made it a bit more user friendly.